ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml

```diff
@@ -0,0 +1,611 @@
+base_images:
+  ansible:
+    name: "4.11"
+    namespace: ocp
+    tag: ansible
+  assisted-image-service:
+    name: ocm-2.5
+    namespace: edge-infrastructure
+    tag: assisted-image-service
+  assisted-installer:
+    name: ocm-2.5
+    namespace: edge-infrastructure
+    tag: assisted-installer
+  assisted-installer-agent:
+    name: ocm-2.5
+    namespace: edge-infrastructure
+    tag: assisted-installer-agent
+  assisted-installer-controller:
+    name: ocm-2.5
+    namespace: edge-infrastructure
+    tag: assisted-installer-controller
+  assisted-service:
+    name: ocm-2.5
+    namespace: edge-infrastructure
+    tag: assisted-service
+  assisted-test-infra:
+    name: ocm-2.5
+    namespace: edge-infrastructure
+    tag: assisted-test-infra
+  aws-ebs-csi-driver-operator-test:
+    name: "4.11"
+    namespace: ocp
+    tag: aws-ebs-csi-driver-operator-test
+  azure-disk-csi-driver-operator-test:
+    name: "4.11"
+    namespace: ocp
+    tag: azure-disk-csi-driver-operator-test
+  base:
+    name: "4.11"
+    namespace: ocp
+    tag: base
+  cnv-ci-src:
+    name: "4.11"
+    namespace: cnv
+    tag: cnv-ci-src
+  cnv-ci-src-upgrade:
+    name: "4.11"
+    namespace: cnv
+    tag: cnv-ci-src-upgrade
+  console-tests:
+    name: "4.11"
+    namespace: ocp
+    tag: console-tests
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  driver-toolkit-tests:
+    name: "4.11"
+    namespace: ocp
+    tag: driver-toolkit-tests
+  gcp-pd-csi-driver-operator-test:
+    name: "4.11"
+    namespace: ocp
+    tag: gcp-pd-csi-driver-operator-test
+  libvirt-installer:
+    name: "4.11"
+    namespace: ocp
+    tag: libvirt-installer
+  upi-installer:
+    name: "4.11"
+    namespace: ocp
+    tag: upi-installer
+  vsphere-csi-driver-operator-test:
+    name: "4.11"
+    namespace: ocp
+    tag: vsphere-csi-driver-operator-test
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.11"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.11"
+  oldest-supported-credentials-request:
+    release:
+      channel: candidate
+      version: 4.11.0-fc.0
+  oldest-supported-z-rollback:
+    release:
+      channel: candidate
+      version: 4.11.0-fc.0
```

Comment:
@wking I saw in `git blame` that you added these to the 4.10 files. Are there reasonable stand-ins for 4.11, or should these be removed (together with the jobs that use them)?

Reply:
no, we should just drop these two and the jobs that consume them until we have an actual 4.11 FC.